src/org.freedesktop.fwupd.xml

```diff
@@ -866,28 +866,28 @@
     </method>
 
     <!--***********************************************************-->
-    <method name='SetBiosAttr'>
+    <method name='SetBiosAttrs'>
```

Comment:
One sec; would it makes sense to use SetBiosAttr() and *also* have CommitBiosAttrs() -- i.e. have an API where you can set A=1, B=2, C=3 and then atomically set all three at the same time? Otherwise I think semantically we need `a(ss)` as a parameter rather than `asas`

Reply:
I don't see a reason to have separate commands for setting and committing. That's not how the kernel works or anything.